filter-claims-data-v1.md

```diff
@@ -25,15 +27,15 @@ Learn how to access [sandbox data]({{ '/access-sandbox-data' | relative_url }})
 {% endcapture %}
 {% include alert.html variant="info" text=versionAlert heading=versionAlertHeading classNames="measure-6" %}
 
-## The _since parameter
+## The \_since parameter
```

Comment:
I'm wondering what this backslash is doing? Not causing an issue-- I'm just curious

Reply:
This was a formatting adjustment that VSCode made when I installed the markdown plugin. It "escapes" special characters (like `_`) which could otherwise be interpreted as the start of an italicized string. In this case, its meant to literally mean an underscore. So by using `\_`, we're telling the markdown compiler (and our future selves), that the character after the backslash is meant to be the "literal" character we're using. [More info here](https://daringfireball.net/projects/markdown/syntax#backslash)